### PR TITLE
Wt1 page3 copy field id fix

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**

### CODE REVIEW

File: walkthroughs-config.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "prettyName": "Red Hat Solution Patterns (OS4)"
+    "prettyName": "Red Hat Solution Patterns-TONY (OS4)"
```
Comment (pb82) [inline walkthroughs-config.json]: this should be removed

Comment (tonyxrmdavidson) [reply walkthroughs-config.json]: Removed with latest commit

Comment (tiffanynolan) [reply walkthroughs-config.json]: The "(OS4)" should also be removed. This was part of a PR that I did recently.

**Review by pb82 — APPROVED** [review]
Verified that the WT is still rendered correctly after adding the ID fields.

**Pull request merged**